Rakefile

```diff
@@ -26,7 +26,7 @@ def run_command(command, opts={})
     FileUtils.cd(path) do
       puts "#{'='*3} #{dir} #{'='*(40 - dir.length)}"
       begin
-        Bundler.clean_system(command)
+        Bundler.unbundled_system(command)
```

Comment:
👍

Reply:
Hm theres a reason we were using clean_system but I've forgotten what it is

Reply:
In my case, I just followed the deprecation warning.

Reply:
Ah then I think its a bundler 1 vs 2 issue

Reply:
Is this a blocker for this PR? 

On another project we added an helper method that check if `unbundled_system` was defined, if not, fallback to `clean_system`. But I think we can keep it like this.

I didn't try all the rake commands and I lack history on the subject so feel free to tell me Jon.

Reply:
No it's our dev tools, we can always revert.